src/lib/route-broadcaster.js

```diff
@@ -55,30 +55,41 @@ class RouteBroadcaster {
         throw e
       }
     }
-    setInterval(() => this.routingTables.removeExpiredRoutes(), this.routeCleanupInterval)
-    defer.setInterval(function * () {
+    this.removeExpiredRoutesSoon()
+    this.broadcastSoon()
```

Comment:
Er, if I misinterpreted the question: I would imagine that the expiry interval would be several times smaller than the broadcast interval, because the former is much cheaper.